src/peft/tuners/lora/config.py

```diff
@@ -76,12 +76,13 @@ class LoraConfig(PeftConfig):
             "the final layer `classifier/score` are randomly initialized and as such need to be trainable and saved."
         },
     )
-    init_lora_weights: bool = field(
-        default=True,
+    init_lora_weights: Optional[str] = field(
```

Comment:
this will lead to tests failing as they are passing boolean flag `False` in many of them.

Reply:
I have changed logics to compare `if init_lora_weights ==True or init_lora_weights == 'zero'` and `elif init_lora_weights  == 'loftq'`